test/unit/location_expansion/mylibrary.rs

```diff
@@ -0,0 +1,3 @@
+pub fn hello_world() {
```

Comment:
Nit: this file can be empty.

Reply:
Well, it can consist of a single newline character, it turns out...